src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs

```diff
@@ -9,12 +9,23 @@
 
 namespace Microsoft.Extensions.Logging.Console
 {
+    /// <summary>
+    /// A logger that writes messages in the console.
+    /// </summary>
     [UnsupportedOSPlatform("browser")]
     internal sealed class ConsoleLogger : ILogger
     {
         private readonly string _name;
         private readonly ConsoleLoggerProcessor _queueProcessor;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConsoleLogger"/> class.
+        /// </summary>
+        /// <param name="name">The name of the logger.</param>
+        /// <param name="loggerProcessor"></param>
```

Comment:
Do these params need summaries still?

Reply:
it's an internal class. I just removed it.